src/rendering/renderers/gpu/renderTarget/GpuRenderTargetSystem.ts

```diff
@@ -376,8 +376,8 @@ export class GpuRenderTargetSystem implements System
                             | GPUTextureUsage.COPY_DST
                             | GPUTextureUsage.RENDER_ATTACHMENT
                             | GPUTextureUsage.COPY_SRC,
-                        format: 'bgra8unorm',
-                        alphaMode: 'opaque',
+                        format: navigator.gpu.getPreferredCanvasFormat(),
```

Comment:
We might want to guard against this property not being available, seems like it will crash in Safari currently:
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/gpu#browser_compatibility

Reply:
This should be ok as to have got this far in the code as it means the `navigator.gpu` has to exist as its required to create the WebGPU device.